src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1194,6 +1213,8 @@ CREATE TABLE user_account
 ,description                 CHARACTER VARYING
 ,user_state                  INTEGER
 ,last_login_ts               TIMESTAMP WITH TIME ZONE
+,is_system_user              BOOLEAN                       NOT NULL

```

Comment:
Will you be updating the standing data scripts for system users?

Reply:
Done in https://github.com/hmcts/darts-api/pull/633

Reply:
i have no list of proposed system users, so no plan currently.
That said i note the standing data script, modernised_darts_standing_data.sql, has been updated to include is_system_user so appears the question has already been addressed